docs/source/gs-processing/developer/input-configuration.rst

```diff
@@ -286,15 +286,15 @@ can contain the following top-level keys:
    feature values in the data.
 -  ``transformation`` (JSON object, optional): The type of
    transformation that will be applied to the feature. For details on
-   the individual transformations supported see :ref:`supported-transformations`.
+   the individual transformations supported see :ref:`gsp-supported-transformations-ref`.
```

Comment:
Do we already create a label about ref:`gsp-supported-transformations-ref`? Did not see it anywhere

Reply:
Yes this was changed in https://github.com/awslabs/graphstorm/pull/735/files so this PR fixes the unresolved refs.